<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ServerException.xml" source-language="en-US" target-language="fr-FR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac50497df7776eb53e36625739cdf9b76066b4b912d.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">0497df7776eb53e36625739cdf9b76066b4b912d</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Runtime.Remoting.ServerException">
          <source>The exception that is thrown to communicate errors to the client when the client connects to non-.NET Framework applications that cannot throw exceptions.</source>
          <target state="translated">Exception levée pour communiquer des erreurs au client lorsque celui-ci se connecte à des applications autres que des applications .NET Framework, qui ne peuvent pas lever d'exceptions.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.Remoting.ServerException">
          <source><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> uses the HRESULT COR_E_SERVER, which has the value 0x8013150E.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> utilise le HRESULT COR_E_SERVER dont la valeur est 0x8013150E.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.Remoting.ServerException">
          <source><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> uses the default <ph id="ph2">&lt;xref:System.Object.Equals%2A&gt;</ph> implementation, which supports reference equality.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> utilise la valeur par défaut <ph id="ph2">&lt;xref:System.Object.Equals%2A&gt;</ph> implémentation, qui prend en charge l’égalité des références.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Runtime.Remoting.ServerException">
          <source>For a list of initial property values for an instance of <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph>, see the <ph id="ph2">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> constructors.</source>
          <target state="translated">Pour obtenir la liste de valeurs de propriété initiales d’une instance de <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph>, consultez la <ph id="ph2">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> constructeurs.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="T:System.Runtime.Remoting.ServerException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> class.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> class with default properties.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> avec des propriétés par défaut.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor">
          <source><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> inherits from the <ph id="ph2">&lt;xref:System.SystemException&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> hérite de la <ph id="ph2">&lt;xref:System.SystemException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor">
          <source>The following table shows initial property values for an instance of the <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> class:</source>
          <target state="translated">Le tableau suivant montre les valeurs initiales des propriétés d’une instance de la <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> classe :</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor">
          <source>Value</source>
          <target state="translated">Value</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor">
          <source>The empty string ("")</source>
          <target state="translated">La chaîne vide (« »)</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source>The message that describes the exception</source>
          <target state="translated">Message décrivant l'exception.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> class with a specified message.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> avec le message spécifié.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> inherits from the <ph id="ph2">&lt;xref:System.SystemException&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> hérite de la <ph id="ph2">&lt;xref:System.SystemException&gt;</ph> classe.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source>The following table shows initial property values for an instance of the <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> class:</source>
          <target state="translated">Le tableau suivant montre les valeurs initiales des propriétés d’une instance de la <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> classe :</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source>Property Type</source>
          <target state="translated">Type de propriété</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source>Condition</source>
          <target state="translated">Condition</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String)">
          <source>The <ph id="ph1">`message`</ph> string</source>
          <target state="translated">Le <ph id="ph1">`message`</ph> chaîne</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Message d'erreur qui explique la raison de l'exception.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Exception ayant provoqué l'exception actuelle.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>InnerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Si le paramètre <bpt id="p1">&lt;c&gt;</bpt>InnerException<ept id="p1">&lt;/c&gt;</ept> n’a pas la valeur <ph id="ph1">&lt;see langword="null" /&gt;</ph>, l’exception actuelle est levée dans un bloc <ph id="ph2">&lt;see langword="catch" /&gt;</ph> qui gère l’exception interne.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialise une nouvelle instance de la classe <ph id="ph1">&lt;see cref="T:System.Runtime.Remoting.ServerException" /&gt;</ph> avec un message d'erreur spécifié et une référence à l'exception interne ayant provoqué cette exception.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Une exception levée en conséquence directe d'une exception précédente doit contenir une référence à celle-ci dans la propriété <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La propriété <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> retourne la même valeur que celle qui a été passée dans le constructeur ou retourne <ph id="ph2">`null`</ph> si la propriété <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> ne fournit pas la valeur d'exception interne au constructeur.</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>The following table shows initial property values for an instance of the <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> class:</source>
          <target state="translated">Le tableau suivant montre les valeurs initiales des propriétés d’une instance de la <ph id="ph1">&lt;xref:System.Runtime.Remoting.ServerException&gt;</ph> classe :</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>Property</source>
          <target state="translated">Property</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>Value</source>
          <target state="translated">Value</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>The inner exception reference</source>
          <target state="translated">La référence à l’exception interne</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Runtime.Remoting.ServerException.#ctor(System.String,System.Exception)">
          <source>The error message string</source>
          <target state="translated">La chaîne du message d’erreur</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>